### PR TITLE
add 1469 DataStorages test

### PR DESCRIPTION
**Pull request by maryvictol**
This PR provides implementation for 1469 DataStorages test.

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesTest.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 package com.epam.pipeline.autotests;
 
 import com.epam.pipeline.autotests.ao.PipelinesLibraryAO;
+import com.epam.pipeline.autotests.ao.ToolTab;
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesTest.java]: Are we still need this?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import static com.epam.pipeline.autotests.ao.Primitive.CLOUD_REGION;
+import static java.lang.String.format;
+
+public class DataStoragesCLITest extends AbstractSinglePipelineRunningTest
+        implements Authorization, Navigation {
+
+    private String storage1 = "dataStorageCLI-" + Utils.randomSuffix();
+    private String storage2 = "dataStorageCLI-" + Utils.randomSuffix();
+    private String pathStorage1 = "";
+    private String pathStorage2 = "";
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java]: Why are not they local?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+import com.epam.pipeline.autotests.utils.Utils;
+import com.epam.pipeline.autotests.utils.listener.Cloud;
+import com.epam.pipeline.autotests.utils.listener.CloudProviderOnly;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static com.epam.pipeline.autotests.ao.Primitive.CLOUD_REGION;
+import static java.lang.String.format;
+
+public class DataStoragesCLITest extends AbstractSinglePipelineRunningTest
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java:35]: @Listeners(value = ConditionalTestAnalyzer.class) should be added

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/StorageContentAO.java [hunk 4]
```diff
@@ -158,6 +158,13 @@ public StorageContentAO rm(String name) {
         return this;
     }
 
+    public StorageContentAO rmFile(String name) {
+        elementRow(name).find(byClassName("ant-btn-danger")).shouldBe(visible).click();
+        final ConfirmationPopupAO<StorageContentAO> popup = new ConfirmationPopupAO<>(this);
+        popup.ensureTitleIs("Remove file").ok();
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/StorageContentAO.java]: It can be combined into one chain

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java:35]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/StorageContentAO.java (hunk 4 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/StorageContentAO.java]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**